exporters/otlp/otlpmetric/otlpmetrichttp/client.go

```diff
@@ -65,14 +65,16 @@ var ourTransport = &http.Transport{
 
 // newClient creates a new HTTP metric client.
 func newClient(cfg oconf.Config) (*client, error) {
+	clonedTransport := ourTransport.Clone()
```

Comment:
If neither TLS nor proxy are configured, we still clone the transport here, which is an unnecessary memory assignment.

Reply:
Thanks @dmathieu, good point.
Committed a fix for that, hopefully clean enough